frontend/src/components/ProjectMemberPanel.vue

```diff
@@ -1,9 +1,70 @@
 <template>
-  <div>
-    <p class="text-lg font-medium leading-7 text-main">
+  <div class="content-center">
+    <span class="text-lg font-medium leading-7 text-main">
       {{ $t("project.settings.manage-member") }}
-    </p>
-    <div v-if="allowAddMember" class="mt-4 w-full flex justify-start">
+    </span>
+
+    <div v-if="allowSyncVCS" class="inline-block float-right">
+      <span class="normal-link text-sm mr-5" @click="openWindowForVCSMember">
+        {{ $t("project.settings.view-vcs-member") }}
+      </span>
+
+      <span class="text-sm text-control">
+        {{ $t("project.settings.sync-from-vcs") }}
+      </span>
+      <heroicons-outline:refresh
+        v-if="project.roleProvider !== 'BYTEBASE'"
```

Comment:
Should this be == GITLAB_SELF_HOST.

Trying to have less negation and scale to other types such as GitHub later.

Reply:
The reason why I take this approach is that if we incorporate more providers in the future, we need nearly no change applied to here.